templates/VALIDATION.md

```diff
@@ -148,6 +149,7 @@ kind: Template
 * `valid`: the rule must be *ignored* if the jsonpath given as value doesn't exist.
 *PLEASE NOTE* that even if values of this key are required to be JSONPaths, you still need to use the `jsonpath::` prefix
 as explained above.
+* `justWarning`: if applying of the rule with this flag fails, it will be only logged into validator logs and validation will not fail.
```

Comment:
"violating rule with justWarning field set will emit a warning only instead of failing the validation"